lib/galaxy_test/api/_framework.py

```diff
@@ -1,3 +1,5 @@
+from unittest import TestCase
```

Comment:
I think that might be not quite right ? Did this maybe go wrong with the rebase ?

Reply:
Actually this got lost during the rebase. If I just use `class ApiTestCase(FunctionalTestCase, UsesApiTestCaseMixin):` instead of class `ApiTestCase(FunctionalTestCase, UsesApiTestCaseMixin, TestCase):` I get 


```
____________________ ERROR collecting test/integration/test_history_import_export.py ____________________
test/integration/test_history_import_export.py:5: in <module>
    class ImportExportHistoryOutputsToWorkingDirTestCase(ImportExportHistoryTestCase, IntegrationTestCase):
E   TypeError: Cannot create a consistent method resolution
E   order (MRO) for bases TestCase, UsesApiTestCaseMixin
```

But maybe there is another solution for this Problem -- but I was unable to find one for days.

Reply:
OK, no, I think that's fine then